### ISSUE TITLE
Should getFilteredData return hidden column data?

### ISSUE THREAD

**Issue by cjohnsonuk** [koalyptus/TableFilter#94]
Can we have a parameter added to getFilteredData to choose whether the data in hidden columns is included in the array returned please?  Hidden rows (though filtering) are not exported but hidden columns appear to be.  

Or have I missed something / am I doing something wrong?

Thanks

**Comment by koalyptus** [comment]
Nothing wrong on your side, the API does not take into account the ``columns visibility`` extension at the moment. 
Tagging this as an enhancement.

**Comment by koalyptus** [comment]
Just published v0.0.22 which implements the ``excludeHiddenCols`` parameter for the ``getTableData`` and ``getFilteredData`` APIs, example:
```
var data = tf.getFilteredData(false, true); // 2nd param excludes hidden cols
```

**Comment by cjohnsonuk** [comment]
I've tested it today and that change works great, thank you.  I've updated the documentation to reflect the changes! 
:-)
